WordPress/Classes/Utility/DisplayableImageHelper.m

```diff
@@ -81,7 +81,6 @@ + (NSString *)searchPostContentForImageToDisplay:(NSString *)content
     // Find all the image tags in the content passed.
     NSArray *matches = [regex matchesInString:content options:0 range:NSMakeRange(0, [content length])];
 
-    NSString *firstImageWithNoSize;
     NSInteger currentMaxWidth = FeaturedImageMinimumWidth;
     for (NSTextCheckingResult *match in matches) {
```

Comment:
My apologies, I missed a couple of things on my first pass. :(  

The for loop here is going to look for the largest image out of all the images in the post content, and then that image will be returned.  We just want the first match now.

There's also one failing test `ReaderPostServiceRemoteTests testFeaturedImageFromDictionary`.

Sorry for drawing things out.  Everything else is 👍